charts/incubator/minecraft-bungeecord/values.yaml

```diff
@@ -43,12 +43,12 @@ service:
         targetPort: 25577
         protocol: HTTP
 rcon:
-    enabled: true
-    ports:
-      rcon:
-        enabled: true
-        port: "{{ .Values.env.RCON_PORT }}"
-        targetPort: "{{ .Values.env.RCON_PORT }}"
+  enabled: true
+  ports:
+    rcon:
+      enabled: true
+      port: "{{ .Values.env.RCON_PORT }}"
+      targetPort: "{{ .Values.env.RCON_PORT }}"
```

Comment:
Without `env.RCON_PORT` available it would fail. (That's why I added it under env)